src/components/RiskClausesLayout.test.tsx

```
@@ -17,12 +18,25 @@ vi.mock("react-icons/fa", () => ({
   FaSpinner: (props: any) => <div data-testid="spinner" {...props} />,
 }));
 
+/* ---------- lucide‑react mock ---------- */
+vi.mock("lucide-react", () => ({
+  Lightbulb: (p: any) => <svg data-testid="lightbulb-icon" {...p} />,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)